sourcecode/hub/app/Http/Controllers/UserController.php

```diff
@@ -55,4 +55,21 @@ public function savePreferences(SavePreferencesRequest $request): RedirectRespon
 
         return to_route('user.preferences');
     }
+
+    public function myAccount(): View
+    {
+        return view('user.my-account');
+    }
+
+    public function saveUsername(StoreUserRequest $request): RedirectResponse
```

Comment:
This should be `saveName`, as this is a real name that we display. There are no usernames in this system.